tensorflow/python/data/experimental/kernel_tests/map_and_batch_test.py

```diff
@@ -399,5 +400,76 @@ def map_fn(x):
       self.evaluate(get_next())
 
 
+class MapAndBatchDatasetSerializationTest(
```

Comment:
Replace "DatasetSerialization" with "Checkpoint", so `MapAndBatchCheckpointTest`

Reply:
addressed in the commit